sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -61,6 +62,11 @@ private[sql] class SharedState(val sparkContext: SparkContext) extends Logging {
       // When neither spark.sql.warehouse.dir nor hive.metastore.warehouse.dir is set,
       // we will set hive.metastore.warehouse.dir to the default value of spark.sql.warehouse.dir.
       val sparkWarehouseDir = sparkContext.conf.get(WAREHOUSE_PATH)
+      logInfo(s"Setting hive.metastore.warehouse.dir ($hiveWarehouseDir) to the value of " +
```

Comment:
when we hit this condition because we provide a `WAREHOUSE_PATH `, maybe the `hive.metastore.warehouse.dir` also have been set